### PR TITLE
FitsNoise.py and new config file

### PR DESCRIPTION
**Pull request by ajwheeler**
This is my FitsNoise type (which now supports adding the background from a fits file), as well as a config file that templates focal.yaml but uses FitsNoise and DES_PSFEx.

### CODE REVIEW

**Comment by ajwheeler** [comment]
I _think_ the right thing to do is merge this with `wrong_wcs`, but I'm not sure.

File: examples/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.fits
+*.fits.fz
```
Comment (rmjarvis) [inline examples/.gitignore]: Minor point, but I'd rather leave the examples/output/.gitignore alone and remove these.  We might want fits files in the repo at some point as inputs.  Ignoring everything in the output directory seems safe though.

File: examples/focal.yaml [hunk 2]
```diff
@@ -309,7 +309,7 @@ output:
     dir: output
     file_name:
         type: FormattedStr
-        format: "sim_%s_%02d.fits.fz"
+        format: "sim_%s_%02d.fits"
```
Comment (rmjarvis) [inline examples/focal.yaml]: Probably don't need this change.  Rice compression is usually good for these...

**Review by rmjarvis — APPROVED** [review]
This is nice.  Other than a couple very minor points, it all looks good to me.

**Pull request merged**